packages/calcite-components/src/components/block-section/block-section.stories.ts

```diff
@@ -0,0 +1,31 @@
+import { html } from "../../../support/formatting";
+
+export default {
+  title: "Components/Block Section",
+};
+
+export const icons_TestOnly = (): string => html`
+  <calcite-block heading="Heading" description="summary" collapsible open>
```

Comment:
Can you move this into the block stories? We usually include child stories to the parent one.